declare function onErrorResumeNext<T extends () => U, U = unknown>(fn: T, context?: undefined): U | undefined;
declare function onErrorResumeNext<T extends (this: V) => U, U = unknown, V = undefined>(fn: T, context: V): U | undefined;

export { onErrorResumeNext };
